### ISSUE TITLE
How to use with an elastic APIKey?

### ISSUE THREAD

**Issue by wolfejw86** [pinojs/pino-elasticsearch#55]
I only see support for the basic user:pass in the url auth mechanism.  Is there an example showing how to use an apikey auth mechanism instead? For example, getting this data to the client:

https://www.elastic.co/guide/en/cloud/current/ec_connect_a_client.html#ec_api_key

**Comment by mcollina** [comment]
cc @delvedor

**Comment by delvedor** [comment]
You are right, currently you can’t pass an api key. It would be nice to expose it as the client does support it.
Would you like to send a pr? :)

**Comment by wolfejw86** [comment]
Okay sure thing, I'd be happy to.

**Comment by wolfejw86** [comment]
@delvedor @mcollina Pull request is up 😄
